File: python/49d_1899A_GamewithIntegers.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        if n % 3 == 0 :
            ans.append("Second")
        else:
            ans.append("First")
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()

File: cpp/49d_1899A_GamewithIntegers.cpp
#include<bits/stdc++.h>

std::string solve(int n){
    if(n % 3 == 0){
        return "Second";
    }
    else{
        return "First";
    }
}

int main(){
    int t, n;
    std::vector<std::string> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>n;
        ans.push_back(solve(n));
    }
    for (auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}